applications/FluidDynamicsApplication/python_scripts/apply_inlet_process.py

```diff
@@ -8,22 +8,20 @@ def Factory(settings, Model):
         raise Exception("expected input shall be a Parameters object, encapsulating a json string")
     return ApplyInletProcess(Model, settings["Parameters"])
 
+def GetHistoricalVariables(settings):
+    copy_settings = settings.__deepcopy__()
+    copy_settings.AddMissingParameters(ApplyInletProcess.GetDefaultParameters())
+    model_part_name = copy_settings["model_part_name"].GetString()
+    root_model_part = model_part_name[:model_part_name.index(".")]
```

Comment:
What happens if `model_part_name` is the root model part, won't this fail at that point raising `ValueError`?

Reply:
you're right, corrected.

Reply:
:+1: